contracts/HydraDelegation/IDelegation.sol

```diff
@@ -11,11 +11,39 @@ interface IDelegation is IWithdrawal {
 
     error InvalidMinDelegation();
 
+    /**
+     * @notice Changes the minimum delegation amount
+     * @dev Only callable by the admin
+     * @param newMinDelegation New minimum delegation amount
+     */
+    function changeMinDelegation(uint256 newMinDelegation) external;
```

Comment:
view and pure functions must be below others of the same type (by type mean external, public, etc.)